themes/bootstrap3/less/components/search.less

```diff
@@ -151,7 +151,7 @@ header .container.navbar { margin-bottom: 0; }
       &.medium img,
       &.large img {
         width: auto;
-        max-width: 100%;
+        max-width: initial;
```

Comment:
What is the intent behind this change? It is causing problems for me:

![image](https://user-images.githubusercontent.com/309069/85715489-82547700-b6b9-11ea-9ad9-f4bd61f1a366.png)

Reply:
(I should note that this record displays correctly when AJAX covers are enabled, but it displays incorrectly as above when the setting is disabled).

Reply:
I probably had to make some mistake, but not sure where. As you mentioned - it works with ajax covers, and that's why I changed it. But i forget to test it without ajax covers...

Reply:
We can probably ask for @crhallberg's assistance on this if necessary; he might have some ideas.

Reply:
I tried to fix it, could you please test it? It is in commit 2a6b4f4

Reply:
Looks good to me!